tests/composite_pk/tests.py

```diff
@@ -158,6 +158,10 @@ def test_query(self):
         users = User.objects.values_list("pk").order_by("pk")
         self.assertNotIn('AS "pk"', str(users.query))
 
+    def test_raw(self):
+        users = User.objects.raw("SELECT * FROM composite_pk_user")
+        self.assertEqual(len(users), 1)
```

Comment:
We should assert that the proper values are assigned to `tenant_id`, `user_id`, and `email` by comparing to `self.user`.

Reply:
Thank you for the review! I updated the code by referring to other test cases.